server/account/account.controller.js

```diff
@@ -1,11 +1,11 @@
-const bcrypt = require("bcrypt");
+const bcrypt = require("bcryptjs");
```

Comment:
Use `bcrypt` module instead of `bcryptjs`.

Reply:
sudo npm i bcrypt => shows error.